contracts/NodeVersionBeacon.cdc

```diff
@@ -354,14 +397,15 @@ pub contract ExecutionNodeVersionBeacon {
 
     init(versionUpdateBuffer: UInt64) {
         /// Initialize contract variables
-        self.ExecutionNodeVersionAdminStoragePath = /storage/ExecutionNodeVersionAdmin
+        self.NodeVersionAdminStoragePath = /storage/NodeVersionAdmin
```

Comment:
Probably should use a more specific path name if possible

Reply:
NodeVersionBeaconAdmin perhaps?

Reply:
maybe `FlowNodeVersionBeaconAdmin`